sql3/parser/ast_test.go

```diff
@@ -914,38 +908,38 @@ func TestSelectStatement_String(t *testing.T) {
 	// 	},
 	// }, `WITH "cte" ("x", "y") AS (SELECT *) VALUES (1, 2), (3, 4)`)
 
-	AssertStatementStringer(t, &parser.SelectStatement{
-		Columns: []*parser.ResultColumn{{Star: pos(0)}},
-		Union:   pos(0),
-		Compound: &parser.SelectStatement{
-			Columns: []*parser.ResultColumn{{Star: pos(0)}},
-		},
-	}, `SELECT * UNION SELECT *`)
+	// AssertStatementStringer(t, &parser.SelectStatement{
```

Comment:
I'm a bit concerned about a lot of these getting commented out at once, because they used to work, so if we're having to comment them out, that feels like a regression?

Reply:
Yeah the parser test may have worked, but no syntax support for union yet....